tests/specification_tests/test_causal_dag.py

```diff
@@ -1,8 +1,15 @@
 import unittest
 import os
 import networkx as nx
-from causal_testing.specification.causal_dag import CausalDAG, close_separator, list_all_min_sep
-from tests.test_helpers import create_temp_dir_if_non_existent, remove_temp_dir_if_existent
+from causal_testing.specification.causal_dag import (
```

Comment:
I've never seen imports with brackets like this

Reply:
Again, this is Black being "uncompromising" and is not something I do conciously. It just autoformats. Perhaps we should add this to the discussion with @bobturneruk about coding style.

Reply:
Agree. Good idea. For now I think it is best to revert to how it was for continuity with the rest of the repository.